### PR TITLE
Issue 192: Have Number Provide Complex Conjugate Message

### PR DESCRIPTION
**Pull request by hemalvarambhia**
Issue #192

Should Number Respond To ComplexConjugate?

### CODE REVIEW

File: src/Math-Tests-Complex/PMComplexTest.class.st [hunk 1]
```diff
@@ -750,6 +750,38 @@ PMComplexTest >> testTimesPolynomial [
 	self assert: (poly * c at: 0) equals: c
 ]
 
+{ #category : #tests }
+PMComplexTest >> testWeCanWriteComplexNumbersWhoRealAndImaginaryPartsAreFractions [
```
Comment (hemalvarambhia) [inline src/Math-Tests-Complex/PMComplexTest.class.st:754]: Whose Real....

**Comment by hemalvarambhia** [comment]
Don't merge yet, @SergeStinckwich . I need to correct a selector name.

File: src/Math-Complex/PMComplex.class.st [hunk 2]
```diff
@@ -403,6 +403,11 @@ PMComplex >> asComplex [
 	^self
 ]
 
+{ #category : #converting }
+PMComplex >> asInteger [
+	^ self.
+]
```
[the remark this answers is not in the document]
Comment (hemalvarambhia) [reply src/Math-Complex/PMComplex.class.st:409]: I'll remove this, and accept quotient of complex numbers aren't possible yet.

[the remark this answers is not in the document]
Comment (hemalvarambhia) [reply src/Math-Complex/PMComplex.class.st]: @SergeStinckwich addressed now. Please review when you have energy and capacity.

File: src/Math-Tests-Complex/PMNumberTest.class.st [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Class {
+	#name : #PMNumberTest,
+	#superclass : #TestCase,
+	#category : #'Math-Tests-Complex'
+}
+
+{ #category : #tests }
```
Comment (hemalvarambhia) [inline src/Math-Tests-Complex/PMNumberTest.class.st:7]: Name these categories better.

File: src/Math-Tests-Complex/PMComplexTest.class.st [hunk 4]
```diff
@@ -750,6 +750,38 @@ PMComplexTest >> testTimesPolynomial [
 	self assert: (poly * c at: 0) equals: c
 ]
 
+{ #category : #tests }
```
Comment (hemalvarambhia) [inline src/Math-Tests-Complex/PMComplexTest.class.st:753]: 'Expressing complex numbers' might be a better name.

**Review by SergeStinckwich — APPROVED** [review]

**Pull request merged**